File: Arrays/Merge Sorted Array/Python/builtin_solution.py
def remove_duplicates(nums):
    unique = sorted(set(nums))
    k = len(unique)

    for i in range(k):
        nums[i] = unique[i]

    return k


# Example usage
nums = [1,1,2]
k = remove_duplicates(nums)
print(k)        # Output: 2
print(nums)     # [1,2,2]

File: Arrays/Merge Sorted Array/Cpp/builtin_solution.cpp
#include <bits/stdc++.h>
using namespace std;

int removeDuplicates(vector<int>& nums) {
    auto it = unique(nums.begin(), nums.end());
    nums.resize(distance(nums.begin(), it));
    return nums.size();
}

int main() {
    vector<int> nums = {1,1,2};
    int k = removeDuplicates(nums);

    cout << k << endl;
    for (int num : nums) cout << num << " ";
}
